src/nls/root/strings.json

```diff
@@ -39,6 +39,7 @@
         "SET_TYPE_TRACKING": "Set Type Tracking",
         "SET_TYPE_ALIGNMENT": "Set Type Alignment",
         "UNGROUP_LAYERS": "Ungroup layers",
+        "REORDER_LAYERS": "Reorder Layers",
```

Comment:
I just realized that I don't understand what these names are used for, since it doesn't seem like they're making their way into Photoshop?